mappings/net/minecraft/entity/damage/DamageSource.mapping

```diff
@@ -1,10 +1,43 @@
 CLASS net/minecraft/unmapped/C_sbxfkpyv net/minecraft/entity/damage/DamageSource
+	COMMENT Represents the source of a damage,
```

Comment:
this wording kinda bothers me, you can't have a singular damage
```suggestion
	COMMENT Represents the source of a damaging event,
```
also considered damage-causing event, both work